go.work

```diff
@@ -1,4 +1,4 @@
-go 1.21
+go 1.21.3
```

Comment:
It didnt seem to work with just the minor, looks like I had to use the patch as well